task/monorail-server/src/bsp/sidecar_bc.rs

```diff
@@ -374,30 +393,21 @@ impl<'a, R: Vsc7448Rw> Bsp<'a, R> {
 
     pub fn phy_vsc8562_init(&mut self) -> Result<(), VscError> {
         if let Some(phy_rw) = &mut self.vsc8562 {
-            // Do a hard power cycle of the PHY; the fine details of sequencing
-            // are handled by the FPGA
-            phy_rw
-                .set_phy_power_enabled(false)
-                .map_err(|e| VscError::ProxyError(e.into()))?;
-            sleep_for(10);
-            phy_rw
-                .set_phy_power_enabled(true)
+            // Request a reset of the PHY.
+            self.seq
+                .reset_front_io_phy()
```

Comment:
It seems a little silly that we smuggle the `OSC_GOOD/VALID` bits _through_ the FPGA into the `sidecar-seq` task.

We could instead track oscillator good/valid in this task and add a parameter to the RPC function, e.g. `fn reset_front_io_phy(full_power_cycle: bool)`.

On the other hand, we need to put those bits in the FPGA for other reasons, so maybe it's fine 🤷🏻‍♂️

Reply:
I should have done a better job documenting this :) Please see https://github.com/oxidecomputer/hubris/pull/1449#discussion_r1253963119. Yes, `monorail-server` does technically have access to this register (because it needs access to the adjacent SMI registers), the expectation is that the sequencer maintains this state (and updates that register).

Reply:
👍🏻  seems fine to me